resources/language/resource.language.en_gb/strings.po

```diff
@@ -974,6 +974,10 @@ msgstr ""
 
 # Unused 30244 to 30399
 
+msgctxt "#30300"
+msgid "Do you want remove watched status of '{}'?[CR][CR]The operation could take effect in 24h, therefore the title may still appear in the list."
```

Comment:
@tweimer can you take a look also to this for english correctness?

Reply:
In my humble opinion, it should be:

` Do you want to remove watched status of '{}'?[CR][CR]The operation could take up to 24h, therefore the title may still appear in the list.
`